### DISCUSSION TITLE
Gradient Texture Oklab Crash

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#278] [! Welcome - readme first!]
testing the Oklab option of GradientTexture crashed the patch. 
this happend on dev, so not sure how mature this option is anyway.

**Comment by pandrr** [comment]
fixed.

i just implemented that yesterday :)

**Reply by franckaubry** [reply]
@pandrr You are implementing this already? 
Yeah! 🥰
